src/AbstractDescriptor.php

```
@@ -46,7 +46,7 @@ public function __construct(
         /** @psalm-suppress MixedAssignment */
         $this->def = $descriptor->def ?? $descriptor->ref ?? $descriptor->src ?? null;
         /** @psalm-suppress MixedAssignment */
-        $this->doc = $descriptor->doc ?? null;
+        $this->doc = $descriptor->doc->value ?? null;
```

PHPStan (2.0.3)

Cannot access property $value on mixed.


(property.nonObject)